build-tests/node-library-build-eslint-test/.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Yes, I've been using `node-library-build-eslint-test` to debug the ESLint rules.  This seemed like the most convenient approach, since it covers both `eslint-config` and `eslint-plugin`.